bliss/predict.py

```diff
@@ -11,7 +17,101 @@ def prepare_image(x, device):
     return x[:, :, :height, :width]
 
 
-def predict(cfg, image, background):
+def crop_plocs(cfg, h, plocs):
+    tiles_to_crop = cfg.encoder.tiles_to_crop
+    tile_slen = cfg.encoder.tile_slen
+    if not cfg.predict.ifSimulated:
```

Comment:
Please use underscores in variables names rather than camel case for consistency